Tests/test_Cluster.py

```diff
@@ -251,7 +251,7 @@ def test_mask_parse(self):
                 [4.1, 2.2, 0.6, 5.5, 0.6],
                 [2.7, 2.5, 0.4, 5.7, 0.2],
             ],
-            numpy.float,
+            numpy.float64,
```

Comment:
As `numpy.float` was a synonym for `float`, I would prefer to replace it with just `float` here. Then effectively the test does not change. Also, it makes the test a bit more robust, as `float` can be different things on different platforms, but `numpy.float64` is always the same.

Reply:
Since this issue is blocking the CI pipeline, I have made your suggested change. 

I think that having tests implicitly change based on platform architecture may cause issues in the long term. An alternative may be to utilize test parametrization to run the test for both `np.float32` and `np.float64`.